paasta_tools/contrib/get_running_task_allocation.py

```diff
@@ -208,12 +212,30 @@ def parse_args() -> argparse.Namespace:
         default="mesos",
         choices=["mesos", "kubernetes"],
     )
+    parser.add_argument(
+        "--namespace-prefix",
+        help="prefix of the namespace to fetch the logs for" \
+	    "Used only when scheduler is kubernetes",
+        dest="namespace_prefix",
+        default="paasta",
+    )
     return parser.parse_args()
 
 
 def main(args: argparse.Namespace) -> None:
     cluster = load_system_paasta_config().get_cluster()
-    info_list = get_task_allocation_info(args.scheduler)
+    if args.scheduler == "mesos":
+        display_task_allocation_info(cluster, args.scheduler, args.namespace_prefix)
+    else:
+        client = KubeClient()
+        all_namespaces = kubernetes_tools.get_all_namespaces(client)
+        matching_namespaces = [n for n in all_namespaces if args.namespace_prefix in n]
```

Comment:
nit: `n.startswith(args.namespace_prefix)` instead of just checking if it's contained, to make it a real prefix match